storm-client/src/jvm/org/apache/storm/daemon/worker/WorkerState.java

```diff
@@ -199,7 +200,6 @@ public WorkerState(Map<String, Object> conf,
         }
         Collections.sort(localTaskIds);
         this.topologyConf = topologyConf;
-        this.topology = ConfigUtils.readSupervisorTopology(conf, topologyId, AdvancedFSOps.make(conf));
```

Comment:
anything specific to this change?

Reply:
I actually updated. Basically mkReceiveQueueMap() function needs to know `this.taskToComponent` which relies on `this.topology`. So we need to move them up.